includes/course-theme/class-sensei-course-theme.php

```diff
@@ -323,13 +328,34 @@ public function add_sensei_theme_body_class( $classes ) {
 		return $classes;
 	}
 
+
+	/**
+	 * Get the version of the active Learning Mode template.
+	 *
+	 * @return string|null Version string in the format of 4-0-2
+	 */
+	public function get_template_version() {
+		global $_wp_current_template_content;
+
+		preg_match( '/sensei-version--(\d+-\d+-\d+)/', $_wp_current_template_content ?? '', $version_matches );
+		return $version_matches[1] ?? null;
+	}
+
 	/**
 	 * Enqueue styles.
 	 *
 	 * @access private
 	 */
 	public function enqueue_styles() {
-		Sensei()->assets->enqueue( self::THEME_NAME . '-style', 'css/learning-mode.css' );
+
+		$version  = $this->get_template_version();
+		$css_file = 'css/learning-mode.' . $version . '.css';
```

Comment:
This will probably change and need to integrate into the template selection feature instead.